### DISCUSSION TITLE
`unauthorized use of script (1:874)`

### DISCUSSION THREAD

**Discussion by fahad787off** [dashingsoft/pyarmor#1190] [Q&A] [answered]
Traceback (most recent call last):
  File "<frozen __main__>", line 4, in <module>
RuntimeError: unauthorized use of script (1:874)

sir when i tried pyarmor with nutika or also when using rft mode same thing

**Comment by jondy** [comment, marked as answer]
Please search "nuitka" in the online documentation to find solutions
